hw/ip/sram_ctrl/dv/cov/sram_ctrl_cov_excl.el

```diff
@@ -0,0 +1,50 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: udij
+// Format Version: 2
+// Date: Wed May  5 10:09:25 2021
+// ExclMode: default
+//==================================================
+CHECKSUM: "3767637819"
+ANNOTATION: "[EXTERNAL] alert_diff_decoder is fully verified by FPV testbench"
+INSTANCE:tb.dut.u_sram_ctrl.gen_alerts[0].u_prim_alert_sender_parity.i_decode_ping
+CHECKSUM: "3767637819"
+ANNOTATION: "[EXTERNAL] alert_diff_decoder is fully verified by FPV testbench"
+INSTANCE:tb.dut.u_sram_ctrl.gen_alerts[0].u_prim_alert_sender_parity.i_decode_ack
+CHECKSUM: "3767637819"
+ANNOTATION: "[EXTERNAL] alert_diff_decoder is fully verified by FPV testbench"
+INSTANCE:tb.dut.u_sram_ctrl.gen_alerts[1].u_prim_alert_sender_parity.i_decode_ping
+CHECKSUM: "3767637819"
+ANNOTATION: "[EXTERNAL] alert_diff_decoder is fully verified by FPV testbench"
+INSTANCE:tb.dut.u_sram_ctrl.gen_alerts[1].u_prim_alert_sender_parity.i_decode_ack
+CHECKSUM: "2966200294"
+ANNOTATION: "[EXTERNAL] prim_lfsr is verified in separate testbench"
+INSTANCE:tb.dut.u_ram1p_sram.u_lfsr
+CHECKSUM: "3927569706"
+ANNOTATION: "[EXTERNAL] prim_prince is verified in separate testbench"
+INSTANCE:tb.dut.u_ram1p_sram.gen_par_scr[0].u_prim_prince
+CHECKSUM: "1684945168 1445297464"
+INSTANCE: tb.dut.u_sram_ctrl
+ANNOTATION: "[UNR] init_q will always be high when sram_scr_init_i.ack is seen"
+Condition 1 "908177468" "(init_q && sram_scr_init_i.ack) 1 -1" (1 "01")
+CHECKSUM: "3557630994 1702182424"
+INSTANCE: tb.dut.u_sram_ctrl.u_prim_sync_reqack_data.u_prim_sync_reqack
+ANNOTATION: "[UNR] FSM only has two states, default state is impossible"
+Block 10 "170878399" ";"
+ANNOTATION: "[UNR] FSM only has two states, default state is impossible"
+Block 18 "1804254986" ";"
+CHECKSUM: "3754679630 2080018642"
+INSTANCE: tb.dut.u_ram1p_sram
+ANNOTATION: "[UNR] addr_cnt_q will always be less than max value when init_req_q is high"
+Condition 1 "1347804513" "(init_req_q && (addr_cnt_q < (Depth - 1))) 1 -1" (2 "10")
+ANNOTATION: "[UNR] init_req_q will always be high when (addr_cnt_q == Depth - 1)"
+Condition 2 "3065646248" "(init_req_q && (addr_cnt_q == (Depth - 1))) 1 -1" (1 "01")
+CHECKSUM: "3557630994 2212754372"
+INSTANCE: tb.dut.u_sram_ctrl.u_prim_sync_reqack_data.u_prim_sync_reqack
+ANNOTATION: "[UNR] FSM only has two states, default state is impossible"
+Branch 1 "814912860" "dst_fsm_cs" (4) "dst_fsm_cs default,-,-"
+ANNOTATION: "[UNR] FSM only has two states, default state is impossible"
+Branch 0 "1492303176" "src_fsm_cs" (4) "src_fsm_cs default,-,-"
```

Comment:
Would you please run the VCS UNR flow for SRAM so that these don't have to be manually found and added?

Reply:
Does this have to be updated, now that we have the UNR exclusions in https://github.com/lowRISC/opentitan/pull/6712?

Reply:
yup - I will update this PR after I look at some nightly results that include the UNR exclusions.